package.json

```diff
@@ -44,6 +49,10 @@
     "webpack-obj-loader": "^1.0.4"
   },
   "jest": {
+    "testEnvironment": "jsdom",
+    "setupFilesAfterEnv": [
+      "<rootDir>/src/setup-jest.ts"
+    ],
```

Comment:
`setupFilesAfterEnv` - Array of files to run after jsdom is intialized and before each test file is executed.
https://jestjs.io/docs/configuration#setupfilesafterenv-array